src/graph_compute/numa_scheduling.h

```diff
@@ -1,45 +1,87 @@
 #ifndef NUMA_SCHEDULING_H_
 #define NUMA_SCHEDULING_H_
-/*
-
 
 #if D1_NUMA
 
 #include <algorithm>
 #include <unordered_set>
 #include "./common.h"
-#include "./update_function.h"
+#include "./concurrent_queue.h"
+#include "./numa_init.h"
 
 #ifndef CHUNK_BITS
   #define CHUNK_BITS 16
+#elif CHUNK_BITS == 0
+  #error "CHUNK_BITS needs to be greater than 0 for D1_NUMA"
 #endif
 
-vid_t logBaseTwoRoundUp(vid_t x) {
-  vid_t power = 0;
-  while ((1 << power) < x) {
-    power++;
-  }
-  return power;
-}
+#ifndef NUMA_WORKERS
```

Comment:
I don't like having several sets of `#ifndef`s and defaults for NUMA_WORKERS -- I'd say either use the ones in `common.h` or here, but not both because then it's just confusing.